board/views.py

```diff
@@ -153,8 +40,12 @@ def __init__(self, game_id):
 
         self.tweets = []
         tweets_id = twitter_wrapper.get_tweets_by_string(self.slug)
```

Comment:
I guess this will result in calling twitter api for each game even if tweets are not displayed on main page, right?

Reply:
Yes, absolutely.